### PR TITLE
feat(Cypress): enabling parallelization Cypress

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Thanks for the help @ojeytonwilliams  and @raisedadead

### CODE REVIEW

**Review by ojeytonwilliams — DISMISSED** [review]
This looks familiar :)

LGTM :+1:

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Cypress
```
Comment (raisedadead) [inline .github/workflows/cypress.yml:1]:
```suggestion
name: Cypress - Pull Request
```

File: .github/workflows/cypress-push.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+name: Cypress
```
Comment (raisedadead) [inline .github/workflows/cypress-push.yml:1]:
```suggestion
name: Cypress - Push
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Also please rename the `cypress.yml` to `cypress-pr.yml`

**Comment by ojeytonwilliams** [comment]
@Sembauke could you commit the rename Mrugesh suggested? We know the tests pass now, so it's not a problem if they never complete on this PR.

**Comment by Sembauke** [comment]
> @Sembauke could you commit the rename Mrugesh suggested? We know the tests pass now, so it's not a problem if they never complete on this PR.

I already comitted them https://github.com/freeCodeCamp/freeCodeCamp/pull/41782/commits/6bbba5f5596eac31a05904a6790406c69333b8d9

**Comment by Sembauke** [comment]
> @Sembauke could you commit the rename Mrugesh suggested? We know the tests pass now, so it's not a problem if they never complete on this PR.

What should I do with the conflict?

**Comment by ojeytonwilliams** [comment]
> What should I do with the conflict?

```
git checkout main
git pull upstream main
git checkout Enabeling-parallel
git merge main 
git push
```
That should prompt you to fix the conflict (if you use VSCode it'll guide you).  Let us know if you run into any problems.

**Review by raisedadead — APPROVED** [review]

**Review by ojeytonwilliams — APPROVED** [review]
LGTM 👍

**Pull request merged**